src/main/java/org/apache/ibatis/migration/commands/BaseCommand.java

```diff
@@ -82,6 +83,15 @@
   protected BaseCommand(SelectedOptions selectedOptions) {
     this.options = selectedOptions;
     this.paths = selectedOptions.getPaths();
+    if (options.isQuiet()) {
+
+      this.printStream = new PrintStream(new OutputStream() {
+        @Override
+        public void write(int b) throws IOException {
```

Comment:
public void write(int b) {} we aren't doing anything here that throws an exception

Reply:
Hmm... that's a checked exception and Intellij put this method in for me. I did remove it and it didn't complain.. odd